src/PrestaShopBundle/Resources/views/Admin/Configure/ShopParameters/TrafficSeo/Blocks/set_up_urls.html.twig

```diff
@@ -0,0 +1,88 @@
+{#**
+ * 2007-2018 PrestaShop
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ *#}
+
+{% trans_default_domain "Admin.Shopparameters.Feature" %}
+
+{% block set_up_urls_configuration %}
+  <div class="col-xl-10">
+    <div class="card">
+      <h3 class="card-header">
+        <i class="material-icons">settings</i> {{ 'Set up URLs'|trans }}
+      </h3>
+      <div class="card-block row">
+        <div class="card-text">
+          <div class="form-group row">
+            {{ ps.label_with_help(('Friendly URL'|trans), ('Enable this option only if your server allows URL rewriting (recommended).'|trans({}, 'Admin.Shopparameters.Help'))) }}
+            <div class="col-sm">
+              {{ form_errors(seoUrlsForm.set_up_urls.friendly_url) }}
+              {{ form_widget(seoUrlsForm.set_up_urls.friendly_url) }}
+            </div>
+          </div>
+
+          <div class="form-group row">
+            {{ ps.label_with_help(('Accented URL'|trans), ('Enable this option if you want to allow accented characters in your friendly URLs. You should only activate this option if you are using non-latin characters ; for all the latin charsets, your SEO will be better without this option.'|trans({}, 'Admin.Shopparameters.Help'))) }}
```

Comment:
For _Enable this option if you want to allow accented characters in your friendly URLs. You should only activate this option if you are using non-latin characters ; for all the latin charsets, your SEO will be better without this option._, it is two strings reunited in one, right? Plus, the mistake has already been taken into account in the Crowdin catalog but there is no space before the semi-colon. :-(